app/src/main/java/com/kickstarter/ui/activities/SurveyResponseActivity.java

```diff
@@ -64,16 +59,11 @@ protected void onCreate(final @Nullable Bundle savedInstanceState) {
       .compose(bindToLifecycle())
       .compose(observeForUI())
       .subscribe(__ -> lazyConfirmationDialog().show());
-
-    this.viewModel.outputs.startProjectActivity()
-      .compose(bindToLifecycle())
-      .compose(observeForUI())
-      .subscribe(this::startProjectActivity);
   }
 
   private boolean handleProjectUriRequest(final @NonNull Request request, final @NonNull WebView webView) {
     this.viewModel.inputs.projectUriRequest(request);
-    return true;
+    return false;
```

Comment:
returning false here lets the request handler know to continue handling project urls